### ISSUE TITLE
sudo pip not recongnized.

### ISSUE THREAD

**Issue by RCCRAFT1** [Marten4n6/EvilOSX#95]
on the requrements.txt step (2nd) pip after sudo was not recognized as  a command. Do you know whats going on

**Comment by eldho-art** [comment]
Yes I too had this problem sudo didn't recognised but I can explain what problem was there - If you are having a Mac you should type "cd" and then type down  the location or the address of the folder " Evil osx " and press enter . After that you can successfully type down the " pip or sudo " command . If you are having Windows or linux - contact the developer or look for solutions in the internet " how to install evil osx " .

**Comment by eldho-art** [comment]
You should also try using pip instead of sudo

**Comment by Wolfoverflow** [comment]
Ok, firstly sudo does not exist on windows, it is the same as running administrator command prompt. Next, for those mac users, you may need to install pip by first running this "curl -sSL https://bootstrap.pypa.io/get-pip.py -o get-pip.py" without the quotation marks then running python get-pip.py to install pip. Do not use sudo unless as a last resort, AND IT IS NOT AN INSTALLER YOU IDIOTS.
